### ISSUE TITLE
qt5-tools-devel package content is different across arch

### ISSUE THREAD

**Issue by yopito** [voidlinux/void-packages#6658]
In particular, file `/usr/lib/cmake/Qt5Designer/Qt5DesignerConfig.cmake` is provided only by x86_64  package:

`x86_64` arch has files in folder `/usr/lib/cmake/Qt5Designer/`:
```
$ xbps-query -M -R -p pkgver,architecture,build-date qt5-tools-devel
qt5-tools-devel-5.8.0_6
x86_64
2017-05-03 06:43 CEST

$ XBPS_TARGET_ARCH=x86_64 xbps-query -M -R  -f qt5-tools-devel | grep cmake | sort
/usr/lib/cmake/Qt5Designer/Qt5DesignerConfig.cmake
/usr/lib/cmake/Qt5Designer/Qt5DesignerConfigVersion.cmake
/usr/lib/cmake/Qt5Designer/Qt5Designer_QQuickWidgetPlugin.cmake
/usr/lib/cmake/Qt5Help/Qt5HelpConfig.cmake
/usr/lib/cmake/Qt5Help/Qt5HelpConfigExtras.cmake
/usr/lib/cmake/Qt5Help/Qt5HelpConfigVersion.cmake
/usr/lib/cmake/Qt5LinguistTools/Qt5LinguistToolsConfig.cmake
/usr/lib/cmake/Qt5LinguistTools/Qt5LinguistToolsConfigVersion.cmake
/usr/lib/cmake/Qt5LinguistTools/Qt5LinguistToolsMacros.cmake
/usr/lib/cmake/Qt5UiPlugin/Qt5UiPluginConfig.cmake
/usr/lib/cmake/Qt5UiPlugin/Qt5UiPluginConfigVersion.cmake
/usr/lib/cmake/Qt5UiTools/Qt5UiToolsConfig.cmake
/usr/lib/cmake/Qt5UiTools/Qt5UiToolsConfigVersion.cmake
```

For arch `aarch64`, no folder `/usr/lib/cmake/Qt5Designer/`:
```
$ XBPS_TARGET_ARCH=aarch64 xbps-query -M -R --repository=https://repo.voidlinux.eu/current/aarch64 -p pkgver,architecture,build-date qt5-tools-devel
qt5-tools-devel-5.8.0_6
aarch64
2017-05-03 02:38 CEST

$ XBPS_TARGET_ARCH=aarch64 xbps-query -M -R --repository=https://repo.voidlinux.eu/current/aarch64 -f qt5-tools-devel | grep cmake | sort
/usr/lib/cmake/Qt5Help/Qt5HelpConfig.cmake
/usr/lib/cmake/Qt5Help/Qt5HelpConfigExtras.cmake
/usr/lib/cmake/Qt5Help/Qt5HelpConfigVersion.cmake
/usr/lib/cmake/Qt5LinguistTools/Qt5LinguistToolsConfig.cmake
/usr/lib/cmake/Qt5LinguistTools/Qt5LinguistToolsConfigVersion.cmake
/usr/lib/cmake/Qt5LinguistTools/Qt5LinguistToolsMacros.cmake
/usr/lib/cmake/Qt5UiPlugin/Qt5UiPluginConfig.cmake
/usr/lib/cmake/Qt5UiPlugin/Qt5UiPluginConfigVersion.cmake
/usr/lib/cmake/Qt5UiTools/Qt5UiToolsConfig.cmake
/usr/lib/cmake/Qt5UiTools/Qt5UiToolsConfigVersion.cmake
```

` armv7` arch: no folder  `/usr/lib/cmake/Qt5Designer/`:
```
$ XBPS_TARGET_ARCH=armv7l xbps-query -M -R  -p pkgver,architecture,build-date qt5-tools-devel
qt5-tools-devel-5.8.0_6
armv7l
2017-05-03 06:45 CEST

$ XBPS_TARGET_ARCH=armv7l xbps-query -M -R  -f qt5-tools-devel | grep cmake | sort
/usr/lib/cmake/Qt5Help/Qt5HelpConfig.cmake
/usr/lib/cmake/Qt5Help/Qt5HelpConfigExtras.cmake
/usr/lib/cmake/Qt5Help/Qt5HelpConfigVersion.cmake
/usr/lib/cmake/Qt5LinguistTools/Qt5LinguistToolsConfig.cmake
/usr/lib/cmake/Qt5LinguistTools/Qt5LinguistToolsConfigVersion.cmake
/usr/lib/cmake/Qt5LinguistTools/Qt5LinguistToolsMacros.cmake
/usr/lib/cmake/Qt5UiPlugin/Qt5UiPluginConfig.cmake
/usr/lib/cmake/Qt5UiPlugin/Qt5UiPluginConfigVersion.cmake
/usr/lib/cmake/Qt5UiTools/Qt5UiToolsConfig.cmake
/usr/lib/cmake/Qt5UiTools/Qt5UiToolsConfigVersion.cmake
```

**Comment by chneukirchen** [comment]
Cannot reproduce this bug, I see Qt5Designer on all.

Is this still an issue?

**Comment by pullmoll** [comment]
No, this was fixed with the update to 5.10.1
